data_loader.py

```diff
@@ -6,107 +6,111 @@
 import lmdb
 import torch
 
+
 def default_loader(path):
     try:
         im = Image.open(path).convert('RGB')
         return im
     except:
-        # print path
-        return Image.new('RGB', (224,224), 'white')
+        print("Image not found in path: {}".format(path))
```

Comment:
This may make the program unnecessarily verbose. Perhaps try
```
from __future__ import print_function
import sys
print(..., file=sys.stderr)
```

Reply:
I don't think showing stder will be less verbose than a one line error message, but no problem